### ISSUE TITLE
macOS Ventura 13.2.1

### ISSUE THREAD

**Issue by go-ski** [mpimd-csc/flexiblas#35]
I am building flexiblas-3.3.0 on macOS 13.2.1 with brew installed cmake version 3.26.0 and all up to date on x86_64-apple-darwin22.3.0.  The gcc is Apple clang version 14.0.0.

make gets to 54% with the error below. Please let me know if this something that I misconfigured or a real issue on macOS.

Thanks,
George

```
[ 54%] Building C object src/CMakeFiles/flexiblas.dir/lapack_interface/wrapper/dsbgvd.c.o
[ 54%] Building C object src/CMakeFiles/flexiblas.dir/lapack_interface/wrapper/dsbgvx.c.o
[ 54%] Building C object src/CMakeFiles/flexiblas.dir/lapack_interface/wrapper/dsbtrd.c.o
[ 54%] Building C object src/CMakeFiles/flexiblas.dir/lapack_interface/wrapper/dsecnd.c.o
/Users/ost/Downloads/flexiblas-3.3.0/src/lapack_interface/wrapper/dsecnd.c:101:54: error: expected expression
double dsecnd(void){ return FC_GLOBAL(dsecnd,DSECND)(void); }
                                                     ^
/Users/ost/Downloads/flexiblas-3.3.0/src/lapack_interface/wrapper/dsecnd.c:125:66: error: expected expression
double flexiblas_real_dsecnd(void){return flexiblas_real_dsecnd_(void);}
                                                                 ^
/Users/ost/Downloads/flexiblas-3.3.0/src/lapack_interface/wrapper/dsecnd.c:155:68: error: expected expression
double flexiblas_chain_dsecnd(void){return flexiblas_chain_dsecnd_(void);}
                                                                   ^
3 errors generated.
make[2]: *** [src/CMakeFiles/flexiblas.dir/lapack_interface/wrapper/dsecnd.c.o] Error 1
make[1]: *** [src/CMakeFiles/flexiblas.dir/all] Error 2
make: *** [all] Error 2
GO2:build ost$ pwd
/Users/ost/Downloads/flexiblas-3.3.0/build
GO2:build ost$ cmake --version
cmake version 3.26.0

CMake suite maintained and supported by Kitware (kitware.com/cmake).
GO2:build ost$ 
```

**Comment by grisuthedragon** [comment]
I found the issue in the code and I will fix it within the next days.

**Comment by grisuthedragon** [comment]
Will be fixed in 3.3.1.

**Comment by go-ski** [comment]
Many thanks! 3.3.1 works!

**Comment by dasergatskov** [comment]
I seems to be getting the same error with the current git snapshot:
```
% git describe --tags 
v3.4.5-1-g75b09519
% git log -1
commit 75b0951923fba529da8a044d578a055f21b0c759 (HEAD -> master, origin/master, origin/HEAD)
Author: Martin Köhler <koehlerm@mpi-magdeburg.mpg.de>
Date:   Thu Feb 6 15:36:32 2025 +0000

    Fix the following issues from GITHUB
    
    - Closes GH #66
    - Closes GH #64
    - Closes GH #63
    Fix in sync
```
This is on MacOS 15.3, Xcode 16.2, and Homebrew (gfortran 14.2.0) on M4 cpu.
I did
```
cmake ../ -DCMAKE_C_FLAGS="-I/opt/homebrew/opt/libomp/include -Xclang -fopenmp"
```
(flags for cmake to find OMP)
and after make:
```
 % gmake V=1                                                                      
[  0%] Built target cscutils_src
[  0%] Built target cscutils
[  3%] Built target flexiblas_netlib_objs
[ 37%] Built target flexiblas_fallback_lapack_objs
[ 37%] Building C object src/CMakeFiles/flexiblas.dir/lapack_interface/wrapper/dsecnd.c.o
/Users/dmitri/src/flexiblas/src/lapack_interface/wrapper/dsecnd.c:65:73: error: expected expression
   65 | double FC_GLOBAL2(dsecnd,DSECND)(void){ return FC_GLOBAL(dsecnd,DSECND)(void); }
      |                                                                         ^
/Users/dmitri/src/flexiblas/src/lapack_interface/wrapper/dsecnd.c:66:73: error: expected expression
   66 | double FC_GLOBAL3(dsecnd,DSECND)(void){ return FC_GLOBAL(dsecnd,DSECND)(void); }
      |                                                                         ^
2 errors generated.
gmake[2]: *** [src/CMakeFiles/flexiblas.dir/build.make:14625: src/CMakeFiles/flexiblas.dir/lapack_interface/wrapper/dsecnd.c.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:845: src/CMakeFiles/flexiblas.dir/all] Error 2
gmake: *** [Makefile:146: all] Error 2
```
